go/pkg/cosmos/txhistory.go

```diff
@@ -19,6 +22,39 @@ import (
 	"github.com/shapeshift/go-unchained/pkg/tendermint/client"
 )
 
+// cursor stores state client side between paginated requests
+type Cursor struct {
+	LastBlockHeight     uint64 `json:"blockHeight"`
+	LastRecipTxID       string `json:"lastRecipTxID"`
+	LastSendTxID        string `json:"lastSendTxID"`
+	TendermintSentPage  uint   `json:"tendermintSenderPage"`
+	TendermintRecvdPage uint   `json:"tendermintReceiverPage"`
+	TotalSends          uint64 `json:"totalSends"`
+	TotalReceives       uint64 `json:"totalReceives"`
+}
+
+// primary history state struct
+type txHistoryState struct {
+	ctx              context.Context
+	address          string
+	pageSize         uint
+	cursor           Cursor
+	cursorSerialized string
+	sends            []client.TxSearchResponseResultTxs
+	recips           []client.TxSearchResponseResultTxs
```

Comment:
Would like to see same verbiage throughout. `sendTxs`, `receiveTxs`, `SendPage`, `ReceivePage`, `TotalReceiveTxs`, `TotalSendTxs`, etc

Reply:
yep waffled on that, cleaned up some of them before I pushed. I'll clean up the rest 👍